content/influxdb/clustered/write-data/best-practices/data-lifecycle.md

```diff
@@ -1,48 +1,74 @@
+Data ingested into InfluxDB must conform to the retention period of the database
+in which it is stored.
+Points with timestamps outside of the retention period are no longer queryable,
+but may still have references maintained in
+[Object storage](/influxdb/clustered/reference/internals/storage-engine/#object-store)
+or the [Catalog](/influxdb/clustered/reference/internals/storage-engine/#catalog),
+resulting in an increase in operational overhead and cost.
+To reduce these factors, it is important to manage the lifecycle of ingested data.
 
-In order to reduce these factors, it is important to manage the lifecycle of ingested data. You can achieve this by following the best practices below:
+Use the following best practices to manage the lifecycle of data in your
+InfluxDB cluster:
 
-- Use a retention policy which is appropriate for your requirements.
-- Tune the garbage collector cutoff period to ensure that data is removed in a timely manner.
+- [Use appropriate retention periods](#use-appropriate-retention-periods)
+- [Tune garbage collection](#tune-garbage-collection)
 
-### Garbage collector tuning
+## Use appropriate retention periods
 
-Once data falls outside of the retention period it can now be removed by the garbage collector service.
+When [creating or updating a database](/influxdb/clustered/admin/databases/#create-a-database),
+use a retention period that is appropriate for your requirements.
+Storing data longer than is required adds unnecessary operational cost to your
+InfluxDB cluster.
 
-There are two variables which tune this process:
+## Tune garbage collection
 
-- `INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF`: the age at which a parquet file not referenced in the catalog becomes eligible for deletion from object storage. The default is `30d`.
-- `INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF`: defines how long to retain the row referencing a parquet file in the catalog once marked for deletion. The default is `30d`.
+Once data falls outside of a database's retention period, the garbage collection
+service can remove all artifacts associated with the data from the Catalog and Object store.
+Tune the garbage collector cutoff period to ensure that data is removed in a timely manner.
+
+Use the following environment variables to tune the garbage collector:
+
+- `INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF`: the age at which Parquet files not
+  referenced in the Catalog become eligible for deletion from Object storage.
+  The default is `30d`.
+- `INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF`: how long to retain rows in the Catalog
+  that reference Parquet files marked for deletion. The default is `30d`.
 
 {{% warn %}}
 **These must NEVER be set to a value less than `3h` (3 hours).**
 
 IOx requires a grace period before files can be removed.
 {{% /warn %}}
```

Comment:
Is this enforced server-side? If not, shouldn't it be?

Reply:
Yeah, this comes from Dom's suggestion [here](https://github.com/influxdata/docs-v2/pull/5369#pullrequestreview-1996750657). After that I also raised an IOx ticket for the garbage-collector and added the guard in too :+1: